omnigibson/prims/xform_prim.py

```diff
@@ -75,10 +75,6 @@ def _post_load(self):
             material.add_user(self)
             self._material = material
 
-        # Optionally set the scale and visibility
```

Comment:
why is this no longer done? is it because we allocate the setting in entity prim / we don't want to override the native scale?

Reply:
Post-discussion outcome: we undo this change, still keep the optional of setting the scale in load_config of XFormPrim. However, we don't pass in scale in the load_config when we are instantiating RigidPrim/ClothPrim for each link in EntityPrim.